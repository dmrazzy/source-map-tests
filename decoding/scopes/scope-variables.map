{
  "version": 3,
  "file": "scope-variables.js",
  "sources": [
    "original_0.js"
  ],
  "mappings": "",
  "names": [
    "global",
    "GLOBAL_CONSTANT1",
    "GLOBAL_CONSTANT2",
    "toplevelFunction1",
    "function",
    "functionScopedVariable1",
    "functionScopedVariable2",
    "functionScopedVariable3",
    "toplevelFunction2",
    "block",
    "blockScopedVariable1",
    "blockScopedVariable2"
  ],
  "scopes": "BCAAA,DCC,BHKFGI,DGCC,CJB,BHGFKA,DF,BCBIK,DKC,CDF,CBB,CBA"
}