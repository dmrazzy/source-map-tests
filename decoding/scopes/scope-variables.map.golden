{
  "file": "scope-variables.js",
  "mappings": [],
  "sources": [
    {
      "url": "original_0.js",
      "content": null,
      "ignored": false,
      "scope": {
        "start": {
          "line": 0,
          "column": 0
        },
        "end": {
          "line": 31,
          "column": 0
        },
        "name": null,
        "kind": "global",
        "isStackFrame": false,
        "variables": [
          "GLOBAL_CONSTANT1",
          "GLOBAL_CONSTANT2"
        ],
        "children": [
          {
            "start": {
              "line": 10,
              "column": 5
            },
            "end": {
              "line": 19,
              "column": 1
            },
            "name": "toplevelFunction1",
            "kind": "function",
            "isStackFrame": true,
            "variables": [
              "functionScopedVariable1",
              "functionScopedVariable2",
              "functionScopedVariable3"
            ],
            "children": []
          },
          {
            "start": {
              "line": 25,
              "column": 5
            },
            "end": {
              "line": 30,
              "column": 1
            },
            "name": "toplevelFunction2",
            "kind": "function",
            "isStackFrame": true,
            "variables": [
              "functionScopedVariable1"
            ],
            "children": [
              {
                "start": {
                  "line": 26,
                  "column": 8
                },
                "end": {
                  "line": 29,
                  "column": 5
                },
                "name": null,
                "kind": "block",
                "isStackFrame": false,
                "variables": [
                  "blockScopedVariable1",
                  "blockScopedVariable2"
                ],
                "children": []
              }
            ]
          }
        ]
      }
    }
  ],
  "ranges": []
}